### PR TITLE
ORLP-386 - All Services should have an interface (Part 1)

### PR DESCRIPTION
**Pull request by MoGUREV**

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/controller/AccountController.java [hunk 1]
```diff
@@ -23,27 +23,27 @@
 public class AccountController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AccountController.class);
-    private final AccountService accountService;
+    private final AccountServiceImpl accountServiceImpl;
```
Comment (voliynyk) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/AccountController.java]: the name of variable should stay the same `accountService`

Comment (MoGUREV) [reply src/main/java/com/softserve/academy/spaced/repetition/controller/AccountController.java]: Okey, one sec

File: src/main/java/com/softserve/academy/spaced/repetition/controller/CardRatingController.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
     public static final int MAX_RATING = 5;
 
     @Autowired
-    private CardRatingService cardRatingService;
+    private CardRatingServiceImpl cardRatingService;
```
Comment (voliynyk) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/CardRatingController.java]: use interface to inject bean

File: src/main/java/com/softserve/academy/spaced/repetition/controller/RegistrationController.java [hunk 3]
```diff
@@ -23,7 +23,7 @@
     @Autowired
     private RegistrationService registrationService;
     @Autowired
-    private AccountVerificationByEmailService verificationService;
+    private AccountVerificationByEmailServiceImpl verificationService;
```
Comment (voliynyk) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/RegistrationController.java]: use interface to inject bean

File: src/main/java/com/softserve/academy/spaced/repetition/security/FacebookAuthUtil.java [hunk 4]
```diff
@@ -28,13 +28,13 @@
     private final AccountRepository accountRepository;
     private final UserRepository userRepository;
     private final AuthorityRepository authorityRepository;
-    private final AccountService accountService;
+    private final AccountServiceImpl accountService;
```
Comment (voliynyk) [inline src/main/java/com/softserve/academy/spaced/repetition/security/FacebookAuthUtil.java]: use interface

File: src/main/java/com/softserve/academy/spaced/repetition/security/GoogleAuthUtil.java [hunk 5]
```diff
@@ -37,14 +37,14 @@
 
     private final AuthorityRepository authorityRepository;
 
-    private final AccountService accountService;
+    private final AccountServiceImpl accountService;
```
Comment (voliynyk) [inline src/main/java/com/softserve/academy/spaced/repetition/security/GoogleAuthUtil.java]: use interface

File: src/main/java/com/softserve/academy/spaced/repetition/security/GoogleAuthUtil.java [hunk 6]
```diff
@@ -37,14 +37,14 @@
...
 
     private final UserService userService;
 
     @Autowired
     public GoogleAuthUtil(AccountRepository accountRepository, UserRepository userRepository,
                           AuthorityRepository authorityRepository,
-                          RememberingLevelRepository rememberingLevelRepository, AccountService accountService, UserService userService) {
+                          RememberingLevelRepository rememberingLevelRepository, AccountServiceImpl accountService, UserService userService) {
```
Comment (voliynyk) [inline src/main/java/com/softserve/academy/spaced/repetition/security/GoogleAuthUtil.java]: use interface

**Pull request merged**